kernelci.org/content/en/docs/org/tsc/votes.md

```diff
@@ -9,14 +9,49 @@ outcome and a more detailed section listing each individual vote.
 
 ## Summary
 
-| Date                      | Motion                               | Status   |
-|---------------------------|--------------------------------------|----------|
-| [2022-06-09](#2022-06-09) | Add TSC member: Denys                | Approved |
-| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi     | Approved |
-| [2021-07-13](#2021-07-13) | Add TSC member: Alice                | Approved |
+| Date                      | Motion                                                          | Status   |
+|---------------------------|-----------------------------------------------------------------|----------|
+| [2021-12-08](#2022-12-08) | Adopt the currently documented TSC rules as a starting point    | Approved |
+| [2021-12-08](#2022-12-08) | Add mgalka to the GitHub admins team                            | Approved |
+| [2021-12-08](#2022-12-08) | List of SysAdmin resources                                      | Approved |
+| [2022-06-09](#2022-06-09) | Add TSC member: Denys                                           | Approved |
+| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi                                | Approved |
+| [2021-07-13](#2021-07-13) | Add TSC member: Alice                                           | Approved |
 
 ## Votes
 
+### 2022-12-08
+
+**Motion: Confirm a list of the components to which each member of the sysadmin subgroup should have access**
+* Result: approved
+* Effective date: 2022-12-08
+* Voting method: regular TSC meeting/Mail
+* Voting members: mgailka, denis, gtucker, spbnick, broonie, alicef, Khilman
+* Vote count: 7/9 voting, 7/7 in favour
+* Links: 
+* [(Docs)2022-12-08 TSC Monthly](https://docs.google.com/document/d/1aoNpi69WRhn1nENbdAY1CPXYVkufbVnG_lFoGfA0aM4/edit#heading=h.g3k9k5jf9amw)
+* [Mail 2022-12-08](https://groups.io/g/kernelci-tsc/message/617)
```

Comment:
Moving this to a sub-level in the list would be better as otherwise it appears at the same level as `Links:`
```suggestion
  * [(Docs)2022-12-08 TSC Monthly](https://docs.google.com/document/d/1aoNpi69WRhn1nENbdAY1CPXYVkufbVnG_lFoGfA0aM4/edit#heading=h.g3k9k5jf9amw)
  * [Mail 2022-12-08](https://groups.io/g/kernelci-tsc/message/617)
```